halo2_proofs/src/poly/evaluator.rs

```diff
@@ -225,6 +241,19 @@ impl<E, F: Field, B: Basis> Evaluator<E, F, B> {
                     }
                     lhs
                 }
+                Ast::DistributePowers(terms, base) => {
+                    let mut acc = vec![F::zero(); ctx.local_chunk_size()];
```

Comment:
I'd prefer to use the existing basis method to ensure we remain basis-agnostic here (even though this happens to be equivalent for all _current_ bases):
```suggestion
                    let mut acc = B::constant_term(ctx.poly_len, ctx.chunk_size, ctx.chunk_index, F::zero());
```